### DISCUSSION TITLE
[example request] @storybook/react-vite + emotion (ts)

### DISCUSSION THREAD

**Discussion by ghtea** [ben-rogerson/twin.examples#121] [General]
I would really appreciate it if you could add example for @storybook/react-vite + emotion (ts)

currently I was using storybook/vite with twin macro
but since "@vitejs/plugin-react": "^4.0.0",
I get following error in build
- `Error: /virtual:/@storybook/builder-vite/vite-app.js: Cannot find module 'twin.macro' from '/virtual:/@storybook/builder-vite'`

so I have tried fixing `.storybook/main.ts` but couldn't solve yet

thanks

**Comment by ben-rogerson** [comment]
Hey there

I've just added a [storybook-emotion-typescript](https://github.com/ben-rogerson/twin.examples/tree/master/storybook-emotion-typescript) example to show you how it's setup in the latest version. 
Let me know if there's still issues.

**Reply by ghtea** [reply]
thanks
I tried this new example and itself works well
but I couldn't solve my issue in my repo and I think it's related to my turborepo and @vitejs/plugin-react or twin.macro
I will try to solve issue again later (maybe in this month) and leave the progress here
